src/mainboard/protectli/vault_adl_n/Kconfig

```diff
@@ -0,0 +1,86 @@
+if BOARD_PROTECTLI_VP32XX
+
+config BOARD_SPECIFIC_OPTIONS
+	def_bool y
+	select BOARD_ROMSIZE_KB_16384
+	select SOC_INTEL_ALDERLAKE_PCH_N
+	select SOC_INTEL_COMMON_BLOCK_HDA_VERB
+	select SOC_INTEL_ENABLE_USB4_PCIE_RESOURCES
```

Comment:
THese resources are actually not needed on ADL-N as the SoC does not have USB4/Thunderbolt4

Reply:
[Removed](https://github.com/Dasharo/coreboot/pull/567/commits/918da71b70c8cbf82572e81f7dab4b0242af7170)